xwiki-rendering-macros/xwiki-rendering-macro-message/src/test/resources/macromessage1.test

```diff
@@ -63,6 +63,8 @@ onSpace
 onWord [possibility]
 endListItem
 endList [BULLETED]
+beginMetaData [[unchanged-content]=[true]]
```

Comment:
This does not make much sense to me. It should be around the content, not after it.